### ISSUE TITLE
GraphBERT pre-training code

### ISSUE THREAD

**Issue by piyush-J** [microsoft/CodeBERT#124]
Thank you for sharing the Graph Code BERT repository. I want to train the Graph Code BERT MLM in a new language. How do I get the pre-training code (with the loss functions implemented for edge prediction and node alignment)? 

Thank you!

**Comment by piyush-J** [comment]
#107
